bundle-test.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+set -e
+
+yell() { echo "$0: $*" >&2; }
+die() { yell "$*"; exit 111; }
+try() { "$@" || die "cannot $*"; }
```

Comment:
Look unused

Reply:
I included it to try to force errors coming from the if statements, found the suggestion on stackoverflow, but I haven't tested if it actually works.